includes/blocks/components/class-progress-bar.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
About the folder name, "components" seems a little confusing for me. Do you think something like "shared" (which we're using in other parts) or "common" would make more sense here?

If we change it, we need to remember to change the namespace in all the parts too.

Reply:
Yeah, I was thinking this could be smaller pieces, not proper blocks, but right now it's pretty much just a shared block so renamed to `shared`.